src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -110,7 +110,12 @@
 
         .datagrid-body {
             .datagrid-row:last-child {
+
                 .datagrid-cell {
+
+                    border-bottom: 1px solid $clr-lighter-midtone-gray;
+                    margin-bottom: -1px;
```

Comment:
Once again, what does this have to do with the column separator? 

Note that the negative margin-bottom was actually tried initially, and it created unwanted scrollbars with various browsers (especially IE). So this is probably breaking a lot of things, unless you tested all the possible cases (rows filling the datagrid, not filling it, scrollable datagrid, flexible height datagrid, no pagination, ...) on every browser we support, and with scrollbars displayed at all times on Mac.